diff --git a/node_modules/react-native-reanimated/lib/reanimated2/core.js b/node_modules/react-native-reanimated/lib/reanimated2/core.js
index 2581315..ceb47ef 100644
--- a/node_modules/react-native-reanimated/lib/reanimated2/core.js
+++ b/node_modules/react-native-reanimated/lib/reanimated2/core.js
@@ -269,6 +269,6 @@ if (!NativeReanimated.useOnlyV1) {
                 error: runOnJS(capturableConsole.error),
                 info: runOnJS(capturableConsole.info),
             };
-            _setGlobalConsole(console);
+            // _setGlobalConsole(console);
         })();
 }
diff --git a/node_modules/react-native-reanimated/src/reanimated2/core.ts b/node_modules/react-native-reanimated/src/reanimated2/core.ts
index 73e6f9d..a72ab56 100644
--- a/node_modules/react-native-reanimated/src/reanimated2/core.ts
+++ b/node_modules/react-native-reanimated/src/reanimated2/core.ts
@@ -307,6 +307,6 @@ if (!NativeReanimated.useOnlyV1) {
         error: runOnJS(capturableConsole.error),
         info: runOnJS(capturableConsole.info),
       };
-      _setGlobalConsole(console);
+      // _setGlobalConsole(console);
     })();
 }
